packages/react/src/spinner/spinner.tsx

```diff
@@ -0,0 +1,53 @@
+import { forwardRef, useEffect, useState } from "react";
+import { clsx } from "@postenbring/hedwig-css/typed-classname";
+
+interface SpinnerPropsInner extends React.HTMLAttributes<HTMLDivElement> {
+  /**
+   * The size of the Spinner
+   *
+   * @default "medium"
+   */
+  size?: "small" | "medium" | "large";
+
+  /**
+   * The tooltip title
+   *
+   * @default ""
+   */
+  title?: string;
+
+  /**
+   * The delay before showing the spinner
+   *
+   * @default 1000
+   */
+  delay?: number;
```

Comment:
Naise 😄 

💭 
Videre kan vi vurdere fade in etter delay. Mulig dette kan også gjøres med ren css.
Dette blokerer **ikke** denne PR'en.

Reply:
Se https://mitchgavan.com/delay-loading-spinners-appearance/

Reply:
Vi kan fortsatt bruke `delay` som en react prop med vilkårlig nummer, vi bare injecter den som en css variabel. Slik som vi gjør i grid og stack

https://github.com/bring/hedwig-design-system/blob/master/packages/css/src/layout/stack/stack.css
https://github.com/bring/hedwig-design-system/blob/master/packages/react/src/layout/stack/stack.tsx#L82